homeassistant/components/lovelace/resources.py

```diff
@@ -84,13 +94,23 @@ def __init__(self, hass: HomeAssistant, ll_config: LovelaceConfig):
 
     async def _process_create_data(self, data: dict) -> dict:
         """Validate the config is valid."""
-        raise NotImplementedError
+        data = self.CREATE_SCHEMA(data)
+        data[CONF_TYPE] = data.pop(CONF_TYPE_WS)
```

Comment:
This is not something that the collection should be aware off. We should pop this out before it gets here.